cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Manage Certificate Authentication for SQL Clients in CockroachDB Dedicated Clusters
+summary: procedures for managing client certificates for dedicated clusters
```

Comment:
There is a Vale test that fails if you have capitalized "Serverless". There isn't the same Vale test for Dedicated (I don't know why). I think the second edit was because I don't think "dedicated cluster" looks right. I guess we are in a state of ambiguity and I bow to your preferences.